library/qs_source/src/std/convert.qs

```diff
@@ -264,5 +264,37 @@ namespace Microsoft.Quantum.Convert {
         );
     }
 
-    export IntAsDouble, IntAsBigInt, ResultAsBool, BoolAsResult, BoolArrayAsInt, IntAsBoolArray, BoolArrayAsBigInt, BigIntAsBoolArray, ResultArrayAsInt, ResultArrayAsBoolArray, BoolArrayAsResultArray, ComplexAsComplexPolar, ComplexPolarAsComplex;
+    /// # Summary
+    /// Converts a given double-precision floating-point number to a string representation with desired precision, rounding if required.
+    ///
+    /// # Input
+    /// ## input
+    /// Double to be converted.
+    /// ## precision
+    /// Non-negative number of digits after the decimal point.
+    /// # Example
```

Comment:
The only theory I have about formatting is adding `///` before the `/// # Example` line. Running formatter from VS Code doesn't do this, but it's the only thing I can see that's different from other lines.